src/bundles/rune/runes_ops.ts

```diff
@@ -1,361 +1,361 @@
+
+  return Rune.of({
+    vertices: new Float32Array(vertexList),
+    colors: new Float32Array(colorList),
+  });
+};
+
+/**
+ * primitive Rune in the rune of a sail
+ * */
+export const getSail: () => Rune = () => {
+  const vertexList: number[] = [];
+  const colorList: number[] = [];
+
+  vertexList.push(0.5, -1, 0, 1);
+  vertexList.push(0, -1, 0, 1);
+  vertexList.push(0, 1, 0, 1);
+
+  colorList.push(0, 0, 0, 1);
+
+  return Rune.of({
+    vertices: new Float32Array(vertexList),
+    colors: new Float32Array(colorList),
+  });
+};
+
+/**
+ * primitive Rune in the rune of a triangle
+ * */
+export const getTriangle: () => Rune = () => {
+  const vertexList: number[] = [];
+  const colorList: number[] = [];
+
+  vertexList.push(1, -1, 0, 1);
+  vertexList.push(0, -1, 0, 1);
+  vertexList.push(0, 1, 0, 1);
+
+  colorList.push(0, 0, 0, 1);
+
+  return Rune.of({
+    vertices: new Float32Array(vertexList),
+    colors: new Float32Array(colorList),
+  });
+};
+
+/**
+ * primitive Rune with black triangle,
+ * filling upper right corner
+ * */
+export const getCorner: () => Rune = () => {
+  const vertexList: number[] = [];
+  const colorList: number[] = [];
+  vertexList.push(1, 0, 0, 1);
+  vertexList.push(1, 1, 0, 1);
+  vertexList.push(0, 1, 0, 1);
+
+  colorList.push(0, 0, 0, 1);
+
+  return Rune.of({
+    vertices: new Float32Array(vertexList),
+    colors: new Float32Array(colorList),
+  });
+};
+
+/**
+ * primitive Rune in the rune of two overlapping
+ * triangles, residing in the upper half
+ * of
+ * */
+export const getNova: () => Rune = () => {
+  const vertexList: number[] = [];
+  const colorList: number[] = [];
+  vertexList.push(0, 1, 0, 1);
+  vertexList.push(-0.5, 0, 0, 1);
+  vertexList.push(0, 0.5, 0, 1);
+
+  vertexList.push(-0.5, 0, 0, 1);
+  vertexList.push(0, 0.5, 0, 1);
+  vertexList.push(1, 0, 0, 1);
+
+  colorList.push(0, 0, 0, 1);
+
+  return Rune.of({
+    vertices: new Float32Array(vertexList),
+    colors: new Float32Array(colorList),
+  });
+};
+
+/**
+ * primitive Rune in the rune of a circle
+ * */
+export const getCircle: () => Rune = () => {
+  const vertexList: number[] = [];
+  const colorList: number[] = [];
+  const circleDiv = 60;
+  for (let i = 0; i < circleDiv; i += 1) {
+    const angle1 = ((2 * Math.PI) / circleDiv) * i;
+    const angle2 = ((2 * Math.PI) / circleDiv) * (i + 1);
+    vertexList.push(Math.cos(angle1), Math.sin(angle1), 0, 1);
+    vertexList.push(Math.cos(angle2), Math.sin(angle2), 0, 1);
+    vertexList.push(0, 0, 0, 1);
+  }
+  colorList.push(0, 0, 0, 1);
+
+  return Rune.of({
+    vertices: new Float32Array(vertexList),
+    colors: new Float32Array(colorList),
+  });
+};
+
+/**
+ * primitive Rune in the rune of a heart
+ * */
+export const getHeart: () => Rune = () => {
+  const vertexList: number[] = [];
+  const colorList: number[] = [];
+
+  const root2 = Math.sqrt(2);
+  const r = 4 / (2 + 3 * root2);
+  const scaleX = 1 / (r * (1 + root2 / 2));
+  const numPoints = 10;
```

Comment:
Was this reversion intentional?

Reply:
This was part of the change to increase rune resolution? My Windows machine has been having some issues with the line endings, so the merge conflict may have been resolved incorrectly.